src/client/src/Stream.c

```diff
@@ -2745,7 +2912,8 @@ STATUS checkStreamingTokenExpiration(PKinesisVideoStream pKinesisVideoStream)
     // In case of failure with get endpoint (i.e. if client state machine fails at auth,
```

Comment:
need to close this paren in the comments

Reply:
added, though it's 3 lines below.